components/builder-jobsrv/src/server/worker_manager.rs

```diff
@@ -212,6 +212,20 @@ impl WorkerMgr {
             // This unwrap is fine, because we just checked our length
             let mut job = jobs.pop().unwrap();
             self.add_integrations_to_job(&mut job);
+            if job.get_project().has_vcs_auth_token() {
+                match bldr_core::integrations::decrypt(
+                    &self.key_dir,
+                    job.get_project().get_vcs_auth_token(),
+                ) {
+                    Ok(secret) => job.mut_project().set_vcs_auth_token(secret),
+                    Err(e) => {
+                        debug!("failed to decrypt vcs auth token, err={:?}", e);
+                        job.set_state(jobsrv::JobState::Pending);
+                        self.datastore.update_job(&job)?;
+                        continue;
```

Comment:
How will this scenario eventually be handled? It seems like we'll just be perpetually trying to decrypt something we can't decrypt; won't that hold up the entire group? What actions can be taken to remediate this?